src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 57 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 0 > C:\Users\VssAdministrator\AppData\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt && testexe -returncode 1
```powershell
At line:1 char:113
+ ... Data\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt && testexe ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 1 > C:\Users\VssAdministrator\AppData\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt && testexe -returncode 1
```powershell
At line:1 char:113
+ ... Data\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt && testexe ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 1 > C:\Users\VssAdministrator\AppData\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt || testexe -returncode 1
```powershell
At line:1 char:113
+ ... Data\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 0 > C:\Users\VssAdministrator\AppData\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt || testexe -returncode 1
```powershell
At line:1 char:113
+ ... Data\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/1.txt || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement '(testexe -returncode 0 && testexe -returncode 1) > C:\Users\VssAdministrator\AppData\Local\Temp\1d33709c-a8e6-4bf3-911a-e8fc0bedd432/3.t
```powershell
At line:1 char:24
+ (testexe -returncode 0 && testexe -returncode 1) > C:\Users\VssAdmini ...
+                        ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```